### PR TITLE
CP2A-170: New repo for C2PA font web components

### PR DESCRIPTION
**Pull request by chriskushmonotype**
# Issue

- [C2PA-170](https://monotype.atlassian.net/browse/C2PA-170)

# Checklist

- [x] **PR labeled** appropriately
- [ ] **Documentation** updated:
  - [ ] Developer documentation (ReadMe files)
  - [ ] Product documentation (Release notes, PDK Guide, Functional Spec, API documents, ...)
- [ ] **Test case(s)** added
  - [ ] Unit Tests

# Notes for Reviewers

Basic repository documents. Please review them in their entirety.

# Verification Instructions

I modified the usual language to refer to GitHub Issues (since this is a public repo) instead of JIRA.

Regarding the repo settings, I can't seem to see them - I have a message in to the Help Desk to see about getting admin privileges, but it's also possible some of these settings work differently on public repos (vs. the private one with which we are most familiar).

> Actively maintained by the @Monotype/driverpdldev team.

[C2PA-170]: https://monotype.atlassian.net/browse/C2PA-170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Notes for Reviewers
+
+<!--  Information to assist code reviewers:                    -->
+<!--    Dependencies or co-reqs (other branches, other repos)  -->
+<!--    Limitations and/or breakage.                           -->
+
+# Verification Instructions
+<!-- Instructions for checking or testing this change. -->
+
+<!-- NOTES on automated checks:                                             -->
```
Comment (brogdonm) [inline .github/pull_request_template.md]: I believe lines 26-39 are not applicable to this repository, please remove.

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+- [Unit Test Coverage](#unit-test-coverage)
+
+This repository contains Web Components for JavaScript which support the L1, L2,
+and L3 experiences for C2PA.
+
+Following are general guidelines for contributing source to this repo.
+
+All communications regarding this repository are governed by the
+[Monotype Contributor Covenant Code of Conduct](https://github.com/Monotype/.github/blob/main/CODE_OF_CONDUCT.md)
+(MCCCoC).
```
Comment (brogdonm) [inline Contributing.md]: Delete the awkward acronym _MCCCoC_. It is not used anywhere else and just adds noise.

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    modify the checked box for that item from `[ ]` to `[x]` to show that it has
+    been handled
+- *Notes for Reviewers*
+  - This section should give a high level description of the PR and how best to
+    inspect the changes
+- *Verification Instructions*
+  - This section should give step-by-step instructions for verifying that the
+    code changes are working
+
+### Merging
```
Comment (brogdonm) [inline Contributing.md]: Should we keep merging to maintainers only?

File: Contributing.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+The `main` branch contains stable, fully-tested code. It should only be merged
+into via CI/CD flows, and only from the `dev` branch or a hot-fix branch.
+
+Merges into `main` **must always** be plain-merged, **never** squashed; see
+[Merging](#merging) for a thorough-going explanation of this rationale.
+
+## Style
+
+### EditorConfig
```
Comment (brogdonm) [inline Contributing.md:105]: Seeing this section made me think the initial setup should include:

- `.gitignore`
- `.gitattributes`
- `.vscode/extensions.json`
    - For now I suppose it should just recommend the `[EditorConfig for VS Code]` extension.
- `.editorconfig`

**Review by brogdonm — CHANGES_REQUESTED** [review]
Just a couple changes in the documentation and should we include some of the other common files to begin with?

**Comment by chriskushmonotype** [comment]
@brogdonm I pulled over things from our template repository.

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+
+# All files
+[*]
+indent_style = space
+
+# Xml files
+[*.xml]
+indent_size = 2
+
+# C# files
```
Comment (brogdonm) [inline .editorconfig]: This repository will contain JavaScript code, for that reason I think it should delete the csharp references and have configurations for JS.

**Review by brogdonm — CHANGES_REQUESTED** [review]
Looking good, I just have one last request for the `.editorconfig`.

File: .editorconfig (hunk 5 above)
Comment (chriskushmonotype) [reply .editorconfig]: @brogdonm Like so?

**Review by brogdonm — APPROVED** [review]
Awesome.

File: .editorconfig (hunk 5 above)
Comment (brogdonm) [reply .editorconfig]: 👍

**Pull request merged**